src/oatpp/core/data/mapping/type/Type.hpp

```diff
@@ -200,7 +200,7 @@ class ObjectWrapper {
     return m_ptr.get() != other.m_ptr.get();
   }
   
-  explicit inline operator bool() const {
+  explicit virtual inline operator bool() const {
```

Comment:
Please remove `virtual`. 
This functionality is not in the context of this issue.
Also, we want to avoid virtual methods in the `ObjectWrapper`.
We want to keep `sizeof` ObjectWrapper as small as possible.